release/version_swap/src/main.rs

```diff
@@ -0,0 +1,61 @@
+/// Helper script to swap the version and build numbers to the ones located in .version and .version-build
+
+use std::fs;
+use std::io::{self, BufRead, Write};
+
+// Function to read the contents of a file into a vector of lines
+fn read_file_lines(path: &str) -> io::Result<Vec<String>> {
+    let file = fs::File::open(path)?;
+    let reader = io::BufReader::new(file);
+    reader.lines().collect()
+}
+
+// Function to write lines to a file
+fn write_file_lines(path: &str, lines: &[String]) -> io::Result<()> {
+    let mut file = fs::File::create(path)?;
+    for line in lines {
+        writeln!(file, "{}", line)?;
+    }
+    Ok(())
+}
+
+// Function to replace the version number and build number in the core.go
+fn replace_version_number(lines: &[String], new_version: &str, new_build_number: &str) -> Vec<String> {
+    lines
+        .iter()
+        .map(|line| {
+            if line.trim().contains("SDKSemverVersion      =") {
```

Comment:
Have a look at: https://gobyexample.com/embed-directive

I believe this would remove the need for this Rust app altogether.